### DISCUSSION TITLE
Рекомендации по изучению сетей

### DISCUSSION THREAD

**Discussion by AlexanderShurnov** [bol-van/zapret#484] [Q&A] [answered]
С точки зрения твоего опыта подскажи, что изучать для максимально эффективного погружения в тему сетей, их устройства, что рекомендуется в в описании zapret?

**Comment by numbereleven-a** [comment, marked as answer]
![79bcac395664cb08c4cc280066b21e3e](https://github.com/user-attachments/assets/06a2a936-3486-4945-919b-e9cffd359c3a)

**Comment by ValdikSS** [comment]
https://censorbib.nymity.ch/

**Comment by vik021** [comment]
Я бы порекомендовал для начала:
https://linkmeup.ru/blog/1188/
